python/cudf/cudf/_lib/sort.pyx

```diff
@@ -22,7 +22,11 @@ from cudf._lib.cpp.sorting cimport (
     rank,
     segmented_sort_by_key as cpp_segmented_sort_by_key,
     sort as cpp_sort,
+    sort_by_key as cpp_sort_by_key,
     sorted_order,
```

Comment:
Do we need to make naming consistent with cpp_…?

Reply:
For some reason the cython wrapper for `sorted_order` is called `order_by` (hence the lack of need to change).